core-processor/src/ext.rs

```diff
@@ -233,6 +234,7 @@ impl IntoExtInfo for Ext {
         let mut pages_data = BTreeMap::new();
         for page in (0..static_pages.0)
             .map(WasmPageNumber)
+            .skip_while(|page| *page < stack_page_count)
```

Comment:
May be it's better to use `stack_page_count.0..static_pages.0`. Check before that stack_page_count <= static_pages

Reply:
Is `static_page_count > static_pages` case possible?

Reply:
> static_page_count > static_pages case possible?
Yes it's possible if contract has wrong information in __gear_end_stack export.

I don't mind